### PR TITLE
hw-6+7

### PR DESCRIPTION
**Pull request by AntonBakaiev**

### CODE REVIEW

File: students/AntonBakaiev/hw-6+7/Task-1.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const enterNumber = prompt('Enter number');
+
+const initialNumber = () => {
+    if (enterNumber == 0) {
```
Comment (Temu4) [inline students/AntonBakaiev/hw-6+7/Task-1.js:4]: Why you use non-strict equality operator here?
```suggestion
    if (enterNumber === 0) {
```

File: students/AntonBakaiev/hw-6+7/Task-1.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        alert('zero');
+    } else if (enterNumber % 2 === 0) {
+        alert('even');
+    } else if ((enterNumber == 1) || (enterNumber % 3 === 0)) {
```
Comment (Temu4) [inline students/AntonBakaiev/hw-6+7/Task-1.js:8]:
```suggestion
    } else if ((enterNumber === 1) || (enterNumber % 3 === 0)) {
```

Comment (Temu4) [reply students/AntonBakaiev/hw-6+7/Task-1.js:8]: And is current logic correct? What would be the output for `enterNumber = 25`? 😄

**Review by Temu4 — CHANGES_REQUESTED** [review]

File: students/AntonBakaiev/hw-6+7/Task-1.js (hunk 2 above)
Comment (AntonBakaiev) [reply students/AntonBakaiev/hw-6+7/Task-1.js:8]: тепер все вірно працює

File: students/AntonBakaiev/hw-6+7/Task-1.js (hunk 1 above)
Comment (AntonBakaiev) [reply students/AntonBakaiev/hw-6+7/Task-1.js:4]: виправив

File: students/AntonBakaiev/hw-6+7/Task-1.js (hunk 2 above)
Comment (Temu4) [reply students/AntonBakaiev/hw-6+7/Task-1.js]: `enterNumber = 49`
`enterNumber = 11`
`enterNumber = 13`
...

**Pull request merged**